### PR TITLE
feat: Removing right sidebar and moving legend out

### PR DESCRIPTION
**Pull request by ashin-czi**
- #3174

### Reviewers
**Functional:** @seve

**Readability:** @seve 

---

## Changes
- add 
    - Legend in top nav bar
- remove 
    - Right sidebar
- modify

### CODE REVIEW

**Comment by ashin-czi** [comment]
@hthomas-czi 

Are we removing the scaled checkbox in scExpression? I didn't see it on Figma
image

**Comment by seve** [comment]
@ashin-czi 
> Are we removing the scaled checkbox in scExpression? I didn't see it on [Figma](https://www.figma.com/file/JS5ThqTBcpAVuBFglAItKf/Gene-Search?node-id=3526%3A591388) so was wondering if we're still supporting it image

Scale is moving from the right sidebar as a checkbox to the left sidebar as a dropdown.
image

File: frontend/src/views/WheresMyGene/components/Beta/index.tsx [hunk 1]
```diff
@@ -1,18 +1,31 @@
...
+
   return (
-    <StyledCallout intent="info" dismissed={false}>
-      This feature is in beta. We would appreciate your feedback, please fill
-      out a{" "}
+    <StyledNotification
+      intent="info"
+      autoDismiss={false}
+      dismissDirection="left"
+      onClose={function () {}}
```
Comment (seve) [inline frontend/src/views/WheresMyGene/components/Beta/index.tsx:16]: we have a noop util somewhere that we use as a stand in for this

File: frontend/src/views/WheresMyGene/components/Beta/index.tsx [hunk 2]
```diff
@@ -1,18 +1,31 @@
...
+      className="elevated"
+    >
+      This feature is in beta. If you have any suggestions or feeback, please{" "}
       <SubmitIssue
         href="https://docs.google.com/forms/d/e/1FAIpQLSde_zIFZPQD2p0ovaX3Pb7lDOajWJCmOeuX4wQ8Z8Ab5NXUjw/viewform"
         target="_blank"
         rel="noopener"
       >
-        quick survey
+        submit an issue on our GitHub
```
Comment (seve) [inline frontend/src/views/WheresMyGene/components/Beta/index.tsx:26]: We should either leave this with the survey copy or switch the link. 
cc: @hthomas-czi

**Review by seve — CHANGES_REQUESTED** [review]

**Review by seve — APPROVED** [review]

**Pull request merged**